Contributing-code-to-Oppia.md

```diff
@@ -61,37 +61,27 @@ We also **strongly recommend** looking through the resources under "Developing O
-
-* [#8015](https://github.com/oppia/oppia/issues/8015) (Refactoring frontend - Services should return domain objects instead of dicts)
-
-* [#8016](https://github.com/oppia/oppia/issues/8016) (Refactoring frontend - The backend api calls in the frontend should only happen through services)
+Welcome! Please make sure to follow the instructions above if you haven't already.
 
-* [#8038](https://github.com/oppia/oppia/issues/8038) (Refactoring frontend services - The backend api calls in the frontend should only happen through services)
+After that, you can choose a good first issue from the [list of good first issues](https://github.com/oppia/oppia/labels/good%20first%20issue). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
 
-* [#10700](https://github.com/oppia/oppia/issues/10700) (Refactor all migrated object factories to model.ts files in angular)
+You can also browse good first issues for each of the core Oppia Web teams to find something you'd enjoy working on:
 
-* [#8668](https://github.com/oppia/oppia/issues/8668) (Documenting the services)
+- Learner and Creator Experience (LaCE): https://github.com/orgs/oppia/projects/3/views/10, typically frontend or full-stack
+- Contributor Dashboard: https://github.com/orgs/oppia/projects/18/views/5, typically full-stack
+- Developer Workflow: https://github.com/orgs/oppia/projects/8/views/10, typically backend or frontend
 
-* [#10474](https://github.com/oppia/oppia/issues/10474) (Make typescript checks strict)
+### Choosing a good first issue
 
-* [#10616](https://github.com/oppia/oppia/issues/10616) (Introduce new @typescript-eslint rules in the codebase)
+When you've found a good first issue you'd like to tackle, please leave a comment on it that:
 
-These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them. For other issues, you may need to be more independent because often times, we don’t know how to solve them either.
+- @-mentions the lead(s) of the corresponding project that the issue falls under (see details for [LaCE](https://github.com/orgs/oppia/projects/3/views/8?pane=info), [Contributor Dashboard](https://github.com/orgs/oppia/projects/18/views/4?pane=info) or [Developer Workflow](https://github.com/orgs/oppia/projects/8?pane=info)), letting them know you'd like to work on it.
+- says which part of the issue you're taking, if the issue is divided into subparts, and
+- describes in more detail how you'd tackle the issue (e.g. explain which file(s) you would modify and what changes you would make). If your explanation makes sense, we'll assign the issue to you.
```

Comment:
can we add one more point. Example see previous merged PRs, because they are great source of learning.  As i also learn too much from that. Your thoughts?

Reply:
Yup sound good to me. It make sense. Will add advice to issues description, in which checkbox is there.